<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial &amp; TCP Config</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Serial Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Port Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="portNameCombo">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Baud Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="baudRateCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Stop Bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="stopBitsCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_24">
              <item>
               <widget class="QLabel" name="label_37">
                <property name="text">
                 <string>Data Bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="dataBitsCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <item>
               <widget class="QLabel" name="label_38">
                <property name="text">
                 <string>Parity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="parityCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QDialogButtonBox" name="buttonBoxSaveConfig">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>TCP Config</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_28">
              <item>
               <widget class="QLabel" name="label_39">
                <property name="text">
                 <string>Tcp/IP</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="TcpEdit">
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_29">
              <item>
               <widget class="QLabel" name="label_40">
                <property name="text">
                 <string>Port</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>62</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="PortEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QDialogButtonBox" name="buttonBoxSaveTCPConfig">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxSaveConfig</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxSaveTCPConfig</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxSaveConfig</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxSaveTCPConfig</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
